content/api/metrics.apib

```diff
@@ -51,14 +51,19 @@ Definitions for terms found in Metrics API
-| `count_spam_seed`                | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Seed messages delivered to the spam folder.                                                         |
+| `count_inbox_seed`               | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> All Seed messages delivered to the inbox.                                                               |
+| `count_spam_seed`                | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> All Seed messages delivered to the spam folder.
+| `count_inbox_seed_private`       | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Private Seed messages delivered to the inbox.                                                               |
+| `count_spam_seed_private`        | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Private Seed messages delivered to the spam folder.    
+| `count_inbox_seed_virtual`       | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Virtual Seed messages delivered to the inbox.                                                               |
+| `count_spam_seed_virtual`        | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Virtual Seed messages delivered to the spam folder.    
 | `count_moved_to_inbox`           | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Panel messages delivered to the spam folder then moved to the inbox.                                |
 | `count_moved_to_spam`            | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Panel messages delivered to the inbox then moved to the spam folder.                                |
 
 Note: The metrics `count_inbox_panel`,`count_spam_panel`,
-`count_inbox_seed`,`count_spam_seed`,`count_moved_to_inbox`,
- and `count_moved_to_spam` are only available to the SparkPost <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> customers. 
+`count_inbox_seed`,`count_spam_seed`,`count_inbox_seed_private`,`count_spam_seed_private`,`count_inbox_seed_virtual`,`count_spam_seed_virtual`,
```

Comment:
If we add a few spaces with the commas, the UI can choose to wrap the text and stop hiding some of it:
![image](https://user-images.githubusercontent.com/5521809/160860511-199b4609-b2a0-4bd1-bd34-bc5d6242b35b.png)